Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -20,23 +19,47 @@
 
 namespace Web {
 
-ResourceLoader& ResourceLoader::the()
+ResourceLoaderConnectorRequest::ResourceLoaderConnectorRequest()
+{
+}
```

Comment:
Ok, so I looked into it a little bit, and I think that since these classes are abstract they have to have their constructor/destructors out of line? Not sure.

Reply:
Sorry, I completely misunderstood your comment the first time - for some reason, I thought you asked to put "= default" in the header file instead of putting it here in the .cpp.
Fixed :)